test/shared/controllers/validators/resolvers/ResolveDateRangeSpec.scala

```diff
@@ -61,6 +61,13 @@ class ResolveDateRangeSpec extends UnitSpec {
         val result = resolveDateRange(validEnd -> validStart)
         result shouldBe Invalid(List(endBeforeStartDateError))
       }
+
+      "passed an end date equal to start date" in {
```

Comment:
Should we consider moving this valid test to the section titled "return no errors" below line 46? This could help consolidate tests related to error-free scenarios.

Reply:
done